presto-main/src/main/java/com/facebook/presto/spiller/BinaryFileSpiller.java

```diff
@@ -51,14 +52,20 @@
     private final Path targetDirectory;
     private final Closer closer = Closer.create();
     private final BlockEncodingSerde blockEncodingSerde;
+    private final AtomicLong spilledDataSize;
 
     private int spillsCount;
     private final ListeningExecutorService executor;
 
-    public BinaryFileSpiller(BlockEncodingSerde blockEncodingSerde, ListeningExecutorService executor, Path spillPath)
+    public BinaryFileSpiller(
+            BlockEncodingSerde blockEncodingSerde,
+            ListeningExecutorService executor,
+            Path spillPath,
+            AtomicLong spilledDataSize)
```

Comment:
I was thinking we would integrate it with the `OperatorStats`, but we can put that off 'til later